src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/AddAzureSqlDatabaseToFailoverGroup.cs

```diff
@@ -44,19 +54,11 @@ public class AddAzureSqlDatabaseToFailoverGroup : AzureSqlFailoverGroupCmdletBas
         /// </summary>
         [Parameter(Mandatory = true,
             ValueFromPipeline = true,
-            HelpMessage = "The Azure SQL Database to be added to the secondary server.")]
+            HelpMessage = "One or more Azure SQL Databases on the Failover Group's primary server to be added to the Failover Group.")]
         [ValidateNotNullOrEmpty]
         public List<AzureSqlDatabaseModel> Database { get; set; }
```

Comment:
Sorry for being (very) unclear in my examples -- the important part is not that we are piping, it is that we are adding multiple databases at the same time. Using a model parameter, you can list all databases on a server, or list all databases in an elastic pool, and add them to a failover group at the same time (without code to select DatabaseName). Adding takes a long time, so adding many at a time is much better than adding one-by-one.

Your point about 'DatabaseName' being better for beginners or people doing tab-completion is well taken, though. We can do that in the future, but I don't want to put us at risk for the next release by making more changes in this pull request.